editoast/src/views/search.rs

```diff
@@ -657,6 +665,7 @@ pub(super) struct SearchResultItemStudy {
 )]
 #[allow(unused)]
 /// A search result item for a query with `object = "scenario"`
+#[derive(serde::Deserialize)]
```

Comment:
Is this necessary?

Reply:
No, I don't know how it ended up there, I must have not seen it wasn't the right place. I'll delete it :)